### PR TITLE
Migrate host address from the host dimension table to host address dimension table

### PR DESCRIPTION
**Pull request by talanknight**

### CODE REVIEW

**Review by tmessi — CHANGES_REQUESTED** [review]
looks good. I just have some minor style related suggestions

File: internal/db/schema/migrations/oss/postgres/25/01_wh_network_address_dimensions.up.sql [hunk 1]
```diff
@@ -63,4 +63,100 @@ begin;
...
+  where
+      whd.host_address not in (select address from wh_network_address_dimension);
+
+  insert into wh_network_address_group (key)
+  select distinct
+    host_address
+  from wh_host_dimension
+    -- There should only be a conflict when adding a key for 'Unknown' since
+    -- that was added in a previous step of this migration.
+  on conflict do nothing;
```
[the remark this answers is not in the document]
Comment (talanknight) [reply internal/db/schema/migrations/oss/postgres/25/01_wh_network_address_dimensions.up.sql:153]: Done.  Thank you!

[the remark this answers is not in the document]
Comment (tmessi) [reply internal/db/schema/migrations/oss/postgres/25/01_wh_network_address_dimensions.up.sql:153]: You changed this for `wh_network_address_group_membership` but not for the inserts into `wh_network_address_group`

[the remark this answers is not in the document]
Comment (talanknight) [reply internal/db/schema/migrations/oss/postgres/25/01_wh_network_address_dimensions.up.sql]: Woops.  Fixed now.

**Review by tmessi — APPROVED** [review]

**Review by irenarindos — COMMENTED** [review]
Just wanted to comment that you'll need to move your migrations to 26- a PR went in yesterday under 25: https://github.com/hashicorp/boundary/tree/main/internal/db/schema/migrations/oss/postgres/25

**Review by mgaffney — APPROVED** [review]

**Pull request merged**